### PR TITLE
Add buggy divide and subtract functions.

### PR DESCRIPTION
**Pull request by soeugene**
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%22https%3A//github.com/axs221%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%7D%2C%20%22https%3A//github.com/soeugene%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%7D%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345356294%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T20%3A24%3A45Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A17%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/soeugene%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A19%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A20%3A16Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A20%3A29Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A22%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%204b6936b0d45ba22461f747dae6362835bf1d7ff9%20main.cpp%2011%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151772144%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Check%20for%20divide%20by%200%22%2C%20%22created_at%22%3A%20%222017-11-17T19%3A37%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/soeugene%22%7D%7D%2C%20%7B%22body%22%3A%20%22test%20comment%22%2C%20%22created_at%22%3A%20%222017-11-17T19%3A40%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/soeugene%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20main.cpp%3AL1-17%22%7D%2C%20%22Pull%20755ce5c7e5cca5084686d5d8cb721c214e955e13%20main.cpp%2023%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151780969%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Indentation%20issue%22%2C%20%22created_at%22%3A%20%222017-11-17T20%3A21%3A11Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T20%3A22%3A52Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T20%3A23%3A09Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T20%3A23%3A31Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A21%3A25Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-11-17T21%3A21%3A41Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars1.githubusercontent.com/u/607938%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/axs221%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20main.cpp%3AL23-33%22%7D%2C%20%22Pull%20755ce5c7e5cca5084686d5d8cb721c214e955e13%20main.cpp%2011%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151800853%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Wrong%20variable%20is%20being%20checked%20for%20divide%20by%20zero%22%2C%20%22created_at%22%3A%20%222017-11-17T22%3A07%3A37Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/soeugene%22%7D%7D%2C%20%7B%22body%22%3A%20%22comment%22%2C%20%22created_at%22%3A%20%222017-11-17T22%3A08%3A03Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/14897377%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/soeugene%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20main.cpp%3AL1-19%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345356294%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345367703%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345368202%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345368268%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345368313%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23issuecomment-345368655%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151772144%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151772710%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151780969%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151781301%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151781348%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151781433%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151792267%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151792312%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151800853%22%2C%20%22https%3A//github.com/soeugene/GerritTest/pull/2%23discussion_r151800928%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/axs221'></a><a href='https://github.com/soeugene'></a>

- [ ] <a href='#crh-comment-Pull 4b6936b0d45ba22461f747dae6362835bf1d7ff9 main.cpp 11'></a> &nbsp;<b><a href='https://github.com/soeugene/GerritTest/pull/2#discussion_r151772144'>File: main.cpp:L1-17</a></b>
- <a href='https://github.com/soeugene'></a> Check for divide by 0
- <a href='https://github.com/soeugene'></a> test comment
- [ ] <a href='#crh-comment-Pull 755ce5c7e5cca5084686d5d8cb721c214e955e13 main.cpp 11'></a> &nbsp;<b><a href='https://github.com/soeugene/GerritTest/pull/2#discussion_r151800853'>File: main.cpp:L1-19</a></b>
- <a href='https://github.com/soeugene'></a> Wrong variable is being checked for divide by zero
- <a href='https://github.com/soeugene'></a> comment
- [x] <a href='#crh-comment-Pull 755ce5c7e5cca5084686d5d8cb721c214e955e13 main.cpp 23'></a> &nbsp;<b><a href='https://github.com/soeugene/GerritTest/pull/2#discussion_r151780969'>File: main.cpp:L23-33</a></b>
- <a href='https://github.com/axs221'></a> Indentation issue
- <a href='https://github.com/axs221'></a> :+1:
- <a href='https://github.com/axs221'></a> :-1:
- <a href='https://github.com/axs221'></a> :+1:
- <a href='https://github.com/axs221'></a> :-1:
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/soeugene/GerritTest/pull/2#issuecomment-345356294'>General Comment</a></b>
- <a href='https://github.com/axs221'></a> :shipit:
- <a href='https://github.com/soeugene'></a> :shipit:
- <a href='https://github.com/axs221'></a> :shipit:
- <a href='https://github.com/axs221'></a> :shipit::gun:

<a href='https://www.codereviewhub.com/soeugene/GerritTest/pull/2?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/soeugene/GerritTest/pull/2?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/soeugene/GerritTest/pull/2?approve=0'></a>&nbsp;<a href='https://github.com/soeugene/GerritTest/pull/2'></a>
<a href='#crh-end'></a>

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 #include <math.h>
 
+int Minus(uint a, uint b)
+{
+  return a-b;
+}
+
+int Divide(float a, float b)
+{
+  return a/b;
```
Comment (soeugene) [inline main.cpp]: Check for divide by 0

Comment (soeugene) [reply main.cpp]: test comment

Comment (soeugene) [reply main.cpp]: 👎

File: main.cpp [hunk 2]
```diff
@@ -11,6 +23,10 @@ void main()
 {
   std::count << "This is as test"
 
+    int a = 5;
```
Comment (shawnaxsom) [inline main.cpp:26]: Indentation issue

**Review by shawnaxsom — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 2 above)
Comment (shawnaxsom) [inline main.cpp:26]: :+1:

Comment (shawnaxsom) [inline main.cpp:26]: :-1:

Comment (shawnaxsom) [inline main.cpp:26]: :+1:

**Comment by shawnaxsom** [comment]
:shipit:

**Comment by soeugene** [comment]
:shipit:

**Comment by shawnaxsom** [comment]
:shipit:

**Comment by shawnaxsom** [comment]
:shipit::gun:

**Comment by shawnaxsom** [comment]
:shipit:

File: main.cpp (hunk 2 above)
Comment (shawnaxsom) [inline main.cpp:26]: :-1:

Comment (shawnaxsom) [inline main.cpp:26]: :+1:

**Comment by shawnaxsom** [comment]
:shipit:

File: main.cpp [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
 #include <math.h>
 
+int Minus(uint a, uint b)
+{
+  return a-b;
+}
+
+int Divide(float a, float b)
+{
+  if (a != 0) {
```
Comment (soeugene) [inline main.cpp:11]: Wrong variable is being checked for divide by zero

Comment (soeugene) [reply main.cpp:11]: comment

**Pull request closed without merging**